.github/workflows/smoke-test.yml

```diff
@@ -12,6 +12,12 @@ jobs:
   validate-podspec:
     runs-on: macos-latest
     steps:
-      - uses: chartboost/chartboost-mediation-ios-actions/adapter-smoke-test@v1
+      - name: Setup Xcode version
```

Comment:
@bwised don't forget to apply the same to the release workflow or the linting will probably fail there

Reply:
Understood.